### PR TITLE
Make I/O scheduler path arch dependent

### PR DESCRIPTION
**Pull request by smitterl**
On s390x first device is 'dasda'. The path to I/O scheduler is now
a global variable whose value is set depending on arch. I made it
global because it's used in many functions in the script.

* Test run on s390x

JOB ID     : 4659cbc4a6bbececc906e757bf1fecad49b50942
JOB LOG    : /root/avocado/job-results/job-2019-09-27T13.06-4659cbc/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.positive.get_blkio_parameter.running_guest.options.none: PASS (9.71 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 11.29 s
JOB ID     : ab5d02279916aa13bf4f0c0998ec004a626429d6
JOB LOG    : /root/avocado/job-results/job-2019-09-27T13.06-ab5d022/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.positive.set_blkio_parameter.running_guest.change_weight.options.current.inside_boundary: PASS (10.16 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 11.72 s
JOB ID     : e2ad8fa8a40a616e26270f14517528b22dd8804f
JOB LOG    : /root/avocado/job-results/job-2019-09-27T13.07-e2ad8fa/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.negative.get_blkio_parameter.running_guest.cgroup.stop: PASS (9.35 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 10.93 s
JOB ID     : b2247311b84c71f31cffe3a0e368b1e06f13015f
JOB LOG    : /root/avocado/job-results/job-2019-09-27T13.07-b224731/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.negative.set_blkio_parameter.running_guest.change_weight.options.live.invalid_value: PASS (9.33 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 10.92 s

* Test run on x86_64

JOB ID     : 670f547df35e232aff077662c597bc9591e6d5bd
JOB LOG    : /root/avocado/job-results/job-2019-09-27T19.07-670f547/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.positive.get_blkio_parameter.running_guest.options.none: PASS (6.63 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 7.81 s
JOB ID     : 95467761e25c6c150e1e7129e91525fac388030c
JOB LOG    : /root/avocado/job-results/job-2019-09-27T19.07-9546776/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.positive.set_blkio_parameter.running_guest.change_weight.options.current.inside_boundary: PASS (6.43 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 7.74 s
JOB ID     : 4b1c2aab9823125b7e75fbf31ec0b3f372990b2c
JOB LOG    : /root/avocado/job-results/job-2019-09-27T19.07-4b1c2aa/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.negative.get_blkio_parameter.running_guest.cgroup.stop: PASS (6.90 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 8.38 s
JOB ID     : ac12fc1d5c444888f29dca7d561ec52d48267908
JOB LOG    : /root/avocado/job-results/job-2019-09-27T19.07-ac12fc1/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.negative.set_blkio_parameter.running_guest.change_weight.options.live.invalid_value: PASS (6.58 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 8.21 s

### CODE REVIEW

**Comment by smitterl** [comment]
@dzhengfy Please review, thank you :)

File: libvirt/tests/src/virsh_cmd/domain/virsh_blkiotune.py [hunk 1]
```diff
@@ -257,17 +272,18 @@ def run(test, params, env):
...
-    with open('/sys/block/sda/queue/scheduler', 'w') as scf:
+    oldmode = re.findall("\\[(.*?)\\]", iosche)[0]
+    with open(schedulerfd, 'w') as scf:
         if 'cfq' in iosche:
             scf.write('cfq')
         elif 'bfq' in iosche:
             scf.write('bfq')
         else:
-            test.fail('Unknown scheduler in "/sys/block/sda/queue/scheduler"')
+            test.fail('Unknown scheduler in %s', schedulerfd)
```
Comment (dzhengfy) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blkiotune.py]: test.fail('Unknown scheduler in %s' % schedulerfd)

Comment (dzhengfy) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blkiotune.py]: Others looks fine to me.

Comment (smitterl) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blkiotune.py]: @dzhengfy Done. Thank you :)

**Pull request merged**